pubspec.yaml

```
@@ -0,0 +1,17 @@
+name: tizen_interop
+description: Provides Dart bindings and utility functions for Tizen native APIs.
+version: 0.0.1
+homepage: https://developer.tizen.org
```

```suggestion
homepage: https://github.com/flutter-tizen/tizen_interop
```
This URL should be more useful for users, if you're going to upload this package to pub.dev.